### DISCUSSION TITLE
Looking for Sparring Partners

### DISCUSSION THREAD

**Discussion by Tyrant7** [SebLague/Chess-Challenge#342] [General]
I’ve been working on my AI for a few days now and am looking for somebody who is willing to trade scripts to help eachother eachother build stronger engines, give feedback, and just overall discuss chess programming.

**Comment by SebLague** [comment]
Hi, you could have a look at the community-run discord server over [here](https://github.com/SebLague/Chess-Challenge/discussions/156), I'm sure you'd have a good chance of finding someone there.

**Reply by Tyrant7** [reply]
I had no idea there was one. Thanks!
